.github/ISSUE_TEMPLATE/problem-report.yml

```diff
@@ -2,13 +2,39 @@ name: 🐞 Problem Report
 description: Tell us about something that's not working the way you expect.
 body:
   - type: input
-    id: version
+    id: self_hosted_version
     attributes:
-      label: Version
+      label: Self-Hosted Version
       placeholder: 21.7.0 ← should look like this (check the footer)
       description: What version of self-hosted Sentry are you running?
     validations:
       required: true
+  - type: input
+    id: operating_system
+    attributes:
+      label: Operating System
+      placeholder: Debian 11 ← should look like this (check /etc/os-release)
+      description: What operating system are you running self-hosted on?
+    validations:
+      required: true
+  - type: input
+    id: docker_version
+    attributes:
+      label: Docker Version
+      placeholder: 20.10.16 ← should look like this (docker --version)
```

Comment:
How about `docker version` instead? It provides more info, including OS/arch, so asking for OS separately may not be necessary:

```
$ docker version
Client:
 Cloud integration: v1.0.22
 Version:           20.10.13
 API version:       1.41
 Go version:        go1.16.15
 Git commit:        a224086
 Built:             Thu Mar 10 14:08:43 2022
 OS/Arch:           darwin/arm64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.6.1 (76265)
 Engine:
  Version:          20.10.13
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.15
  Git commit:       906f57f
  Built:            Thu Mar 10 14:05:37 2022
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.5.10
  GitCommit:        2a1d4dbdb2a1030dc5b01e96fb110a9d9f150ecc
 runc:
  Version:          1.0.3
  GitCommit:        v1.0.3-0-gf46b6ba
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
$
```

Reply:
I don't see anything useful expect version of docker itself, can you please elaborate on this?

Also we do not support anything expect amd64 architectures, so `OS/Arch` will only tells us about Linux / Windows / Apple (?). I don't have a windows or apple system to test this, mine says `linux/amd64`.

But thinking about this again, operating system helps us in some circumstances, but not that much. So we can keep it out of this PR and add it later in some another PR if we felt the necessity.

Reply:
What do you think about replacing operating system with `docker info --format '{{.Architecture}}'` output?

I don't really see anything useful in `docker version` expect version and cpu architecture.

Reply:
I mean something like this: https://github.com/getsentry/self-hosted/pull/1497/commits/6b0b4ec

Reply:
Yeah that should be OK.